scripts/metadata-export.md

```diff
@@ -0,0 +1,42 @@
+# Metadata-Export script
```

Comment:
add a link back to the list of the scripts (something similar is done in each individual endpoint file toward the endpoint.md list)